spec/requests/v0/profile/addresses_spec.rb

```diff
@@ -162,16 +146,12 @@
     end
   end
 
-  describe 'PUT /v0/profile/addresses' do
-    before do
-      allow(Flipper).to receive(:enabled?).with(:remove_pciu, instance_of(User)).and_return(false)
-    end
-
-    let(:address) { build(:va_profile_address, vet360_id: user.vet360_id) }
+  describe 'PUT /v0/profile/addresses v2' do
```

Comment:
Should it be this, instead?
```suggestion
  describe 'PUT /v2/profile/addresses' do
```

Reply:
oh 🤔 never mind. I guess not. I see `put('/v0/profile/addresses'...` used in tests.